node/_errors.ts

```diff
@@ -74,7 +72,7 @@ function addNumericalSeparator(val: string) {
 export function hideStackFrames(fn: GenericFunction) {
   // We rename the functions that will be hidden to cut off the stacktrace
   // at the outermost one.
-  const hidden = nodeInternalPrefix + fn.name;
+  const hidden = "__node_internal_" + fn.name;
```

Comment:
Changed because following error occured

```zsh
AssertionError: Failed assertion: error: Uncaught ReferenceError: Cannot access 'nodeInternalPrefix' before initialization
  const hidden = nodeInternalPrefix + fn.name;
                 ^
    at hideStackFrames (file:///Users/tagawahirotaka/Desktop/deno_std/node/_errors.ts:77:18)
    at file:///Users/tagawahirotaka/Desktop/deno_std/node/_validators.ts:39:30
```

Reply:
I guess the initialization of that const might be blocked by top-level-await somewhere in the dependencies? Looks like a very strange situation...

Thank you for the fix!

Reply:
>I guess the initialization of that const might be blocked by top-level-await somewhere in the dependencies? Looks like a very strange situation...

I cloudn't figure out... I'll look into it again.